apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformer.java

```diff
@@ -223,7 +416,58 @@ private PreAdjFissClaim transformClaim(FissClaim from, DataTransformer transform
             PreAdjFissClaim.Fields.stmtCovToDate,
             from::hasStmtCovToCymd,
             from::getStmtCovToCymd,
-            to::setStmtCovToDate);
+            to::setStmtCovToDate)
+        .copyEnumAsString(
+            PreAdjFissClaim.Fields.lobCd, true, 1, 1, lobCd.getEnumString(from), to::setLobCd)
+        .copyEnumAsString(
+            PreAdjFissClaim.Fields.servTypeCd,
+            true,
+            1,
+            1,
+            servTypeCd.getEnumString(from),
+            value -> {
+              to.setServTypeCdMapping(PreAdjFissClaim.ServTypeCdMapping.Normal);
+              to.setServTypeCd(value);
+            })
+        .copyEnumAsString(
+            PreAdjFissClaim.Fields.servTypeCd,
```

Comment:
are all of these (lines 422-461) supposed to be `PreAdjFissClaim.Fields.servTypeCd`?

Reply:
Yes they are all referring to the same field by design.  Each of them is one possible outcome of the 4-way oneof field serv_typ_cd.  You'll notice that each of them pulls from a different field in the GRPC FissClaim object.

````protobuf
  // Bill Classification
  oneof serv_typ_cd {
    FissBillClassification serv_type_cd_enum = 44;
    FissBillClassificationForClinics serv_type_cd_for_clinics_enum = 45;
    FissBillClassificationForSpecialFacilities serv_type_cd_for_special_facilities_enum = 46;
    string serv_typ_cd_unrecognized = 47;
  };
````